generator/api/src/test/java/io/fabric8/maven/generator/api/FromSelectorTest.java

```diff
@@ -97,26 +97,92 @@ public void simpleJava11() {
         };
 
         for (int i = 0; i < data.length; i += 4) {
-            prepareExpectionJava11((String) data[i + 2], (RuntimeMode) data[i], (OpenShiftBuildStrategy) data[i + 1]);
+            prepareExpectionJava11Release((String) data[i + 2], (RuntimeMode) data[i], (OpenShiftBuildStrategy) data[i + 1]);
             FromSelector selector = new FromSelector.Default(ctx, "test");
 
             assertEquals(data[i + 3], selector.getFrom());
         }
     }
 
-    private Expectations prepareExpectionJava8(final String version, final RuntimeMode mode, final OpenShiftBuildStrategy strategy) {
-        return prepareExpectation(version, mode, strategy, String.valueOf(8));
+
+    @Test
+    public void Java8BaseImageTarget() {
+        final Object[] data = new Object[] {
```

Comment:
This test seems to be exact copy of `Java8BaseImage` apart from the maven property being set. Can we please refactor common parts or maybe combine these two tests into one?